src/index.js

```diff
@@ -1,17 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <App />,
   document.getElementById('root'),
```

Comment:
[OPTIONAL]

- What do you think about bringing back `React.StrictMode`? Strict mode helps with identifying components with unsafe lifecycles and some other checks during development. To know more about strict mode, please, got through its [documentation](https://reactjs.org/docs/strict-mode.html) 😀